js/src/features/project/types.d.ts

```diff
@@ -89,15 +89,40 @@ declare module "ProjectModels" {
   export interface ProjectTeam {
```

Comment:
Why do you use `interface`. In whole project we use `type`.

Reply:
It applies to all interfaces in your pull request.

Reply:
fix